dateparser/date.py

```diff
@@ -14,6 +14,12 @@
 from dateparser.languages.detection import AutoDetectLanguage, ExactLanguages
 
 
+APOSTROPHE_LOOK_ALIKE_CHARS = [
+    u'\u2019', u'\u02bc', u'\u02bb', u'\u055a', u'\ua78c',
+    u'\uff07', u'\u2032', u'\u2035', u'\u02b9', u'\uff07',
```

Comment:
Agreed with @redapple `\uxxxx` is handy in comments. Also agrees with @seagatesoft's argument that using unicode strings are more distinct than *look alike* literal characters. Unicode names look great -- they might appear a little verbose yet suits our purpose. +1 for @asadurski's above representation.